inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/InlongGroupServiceImpl.java

```diff
@@ -382,6 +382,10 @@ public void updateAfterApprove(InlongGroupApproveRequest approveRequest, String
             entity.setInlongGroupId(groupId);
             entity.setInlongClusterTag(approveRequest.getInlongClusterTag());
             entity.setStatus(GroupStatus.APPROVE_PASSED.getCode());
+            if (approveRequest.getDataReportType() != null
+                    && approveRequest.getDataReportType() != entity.getDataReportType()) {
```

Comment:
It seems that the second condition is unnecessary.

Reply:
It doesn't matter if it is written in this way, and unnecessary modification is reduced: if it has not been changed, the original value will not be modified.

Reply:
I modified the comparison method according to the tooltip